src/types/json.rs

```diff
@@ -242,7 +278,7 @@ where
 ///     let app = App::new().service(
 ///         web::resource("/index.html")
 ///             .app_data(
-///                 // change json extractor configuration
+///                 // Json extractor configuration for only the `Info` struct.
```

Comment:
IMHO it would read better like that:

```suggestion
///                 // JSON extractor configuration for the `Info` struct only.
```